reference/ams-configuration.md

```diff
@@ -30,22 +30,46 @@ AMS provides various configuration items to customize its behavior. The followin
 
 Some features can be conditionally enabled via AMS.
```

Comment:
Move this line after the new content (because we're first talking about ALL features and then about SOME).